cms/templates/cms/benefits_page.html

```diff
@@ -49,6 +49,17 @@ <h1 class="title">
         {% endblock %}
         </div>
       </div>
+      {% if '/organizations/' in request.path %}
+        <div class="mdl-cell mdl-cell--4-col">
```

Comment:
Can we use a `sticky-top` class here, like the one we have in xPRO?

https://github.com/mitodl/mitxpro/blob/master/cms/templates/partials/subnav.html#L1